igvm/drbd.py

```diff
@@ -274,7 +281,7 @@ def wait_for_sync(self):
             else:
                 # Exit the loop if status for current device can't be found
                 show_progress = False
-            sleep(1)
+            sleep(5)
```

Comment:
Wouldn't it be better to retry more times but with shorter sleep? Things displaying nothing for 5 seconds scare people ("my ssh froze", "network is broken").